data-preprocessing/JSON  Section/README.md

```diff
@@ -0,0 +1,12 @@
+# How to use
+
+## Useful Information
+The json files are of two forms. The structure of the Algebra.json is slightly different from Chemistry 2e.json and College Physics 2e.json. As such slightly different block of code handle the return of the sections to account for these differences.
+
+## Function
+The main.py file accepts a path to json files of parsed books. These files contain portions called sections. The file then returns all sessions alongside their page numbers in a csv file format; csv file for each json file.
```

Comment:
Do CSVs work well? @wsedor you have any thoughts on this? Would need to consult QA team as well